src/main.zig

```diff
@@ -162,6 +162,13 @@ pub fn main() anyerror!void {
         return mainArgs(gpa_tracy.allocator(), arena, args);
     }
 
+    // On WASI, the CWD and absolute paths are emulated w.r.t. the available preopens.
+    // Here we initialize that machinery and set the CWD to ".", where the user is
+    // expected to map their current directory using "--dir=."
+    if (builtin.os.tag == .wasi) {
+        try std.os.initPreopensWasi(std.heap.page_allocator, ".");
```

Comment:
Yeah, it is :+1:

I was trying to avoid the dependency on #11053 for now, but whichever PR lands last will need to update this to `os.initPreopensWasi(std.heap.page_allocator, "/preopens/cwd")`

Reply:
Correction: https://github.com/WebAssembly/wasi-filesystem/issues/24#issuecomment-705621285 `.` and ´/` can be mapped differently: "(Ex: In Wasmer executables installed by the package manager come with a manifest; the author can request / and . to be mapped, creating a POSIX-like environment.)"